core/appHandler.js

```diff
@@ -224,7 +225,8 @@ function listUsersAPIRating1(res) {
 }
 
 module.exports.listUsersAPI = function (req, res) {
-	var securityRating = req.params.securityRating ? req.params.securityRating : 0;
+	var vulnKey = 'a3_sensitive_data';
```

Comment:
This is one of the downsides: the final securityRating handler needs to know which vulnerability it's associated with. I couldn't really think of a better way to do it except hardcode the specific key associated with the vulnerability.